sources/web/datalab/polymer/test/table-preview-test.ts

```diff
@@ -74,13 +74,6 @@ describe('<table-preview>', () => {
     Polymer.dom.flush();
   });
 
-  it('displays an empty element if no table is provided', () => {
-    assert(!testFixture.$.placeholder.hidden, 'placeholder should be shown');
```

Comment:
Was this test not working? Should it be fixed and moved somewhere else instead of being deleted?

Reply:
I deleted it because I think it shouldn't be testing this; neither is there a placeholder in table-preview now, nor should the container be hidden any more.
The placeholder should ideally be tested in a preview-pane-test, which we don't have yet.